extensions/opentelemetry/runtime/src/main/java/io/quarkus/opentelemetry/runtime/logs/OpenTelemetryLogHandler.java

```diff
@@ -43,6 +43,7 @@ public void publish(LogRecord record) {
         final LogRecordBuilder logRecordBuilder = openTelemetry.getLogsBridge()
                 .loggerBuilder(INSTRUMENTATION_NAME)
                 .build().logRecordBuilder()
+                .setTimestamp(record.getInstant())
```

Comment:
Needs to be current time, not the record instant, that's the same as the observed from the line bellow.